File: Aula 09/Exemplo3B.java
import java.util.Scanner;

public class Exemplo3B {
    public static void main(String[] args) {
        double salario[] = new double[5];
        int i;
        Scanner sc = new Scanner(System.in);
        for(i=0; i<5; i++){
            System.out.print("Digite o salário do " +(i+1) +"º funcionário: ");
            salario[i] = sc.nextDouble();
        }
        System.out.println("Salário do 4º funcionário antes do método: " + salario[3]);
        calculaAumento(salario, 3);
        System.out.println("Salário do 4º funcionário antes do método: " +salario[3]);
    }
    public static void calculaAumento(double salario[], int i){
            salario[i] = salario[i]*1.03;
    }
}

File: Aula 09/Exemplo3B.py
def calculaSalario(salario, i):
    salario[i] = salario[i] * 1.03
    
salario = []
for i in range(5):
    leia = float(input(f'Digite o salário do {i+1}º funcionário >_ '))
    salario.append(leia)

print(f'Salário do 4º funcionário antes do método: {salario[3]}')
print(salario)
calculaSalario(salario, 3)
print(f'Salário do 4º funcionário após o método: {salario[3]}')
print(salario)